destream/archive.py

```diff
@@ -41,43 +43,46 @@ def __init__(self, name, fileobj=None, source=None, closefd=True):
         else:
             self._decompressors = [type(self)]
             self.compressions = []
-        if hasattr(self, '_compression'):
+        if hasattr(self, "_compression"):
             self.compressions += [self._compression]
 
     @classmethod
-    def _check_availability(self):
+    def _check_availability(cls):
         pass
 
     @classmethod
     def _guess(cls, mime, name, fileobj):
```

Comment:
That one did not get converted to `self` but the previous did?  Was that black or you did it manually?

Reply:
`_guess(cls …` was like that since your edit in 2015. `_check_availability(cls …` was my manual change now. `black` was not involved in either case.

Reply:
Mistakes were made